mod-data-export/src/test/resources/firebird/dataexport/features/delete-job-execution.feature

```diff
@@ -37,8 +38,17 @@ Feature: Test removing job execution
     And match response == {jobExecutionId: '#uuid', jobExecutionHrId: '#number'}
     And def jobExecutionId = response.jobExecutionId
 
+    #should return job execution by id and wait until the job status will be 'COMPLETED'
+    Given path 'data-export/job-executions'

```

Comment:
Wait until export will be finished and the job execution will be created before running the next test.